server/spec/requests/api/v1/users_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+RSpec.describe Api::V1::UsersController do
+  before(:all) do
+    Rails.application.load_seed
+  end
+  #describe "Test Users", :type => :controller do
+  it 'Test we get a success response to GET' do
+    get "/api/v1/users"
+
+    json = JSON.parse(response.body)
+    print(json)
+    # test for the 200 status-code
+    expect(response).to be_success
+  end
+
```

Comment:
I think tests for show e.g. api/v1/users/1 is missing. Can you please add such tests?
Ideas for scenarios to test:

1. with an existing user ID - HTTP Status Code should be 200 (OK) with the response containing the requested user
1. with ID which doesn't correspond to an existing user (expectation would be to return 404 Not Found HTTP status code)